.travis.yml

```diff
@@ -22,9 +22,9 @@ env:
     # Custom environment variables for the ARM wheel builder
     - CIBW_BUILD_VERBOSITY=1
     - CIBW_TEST_COMMAND="bash {project}/build_tools/travis/test_wheels.sh"
-    - CIBW_ENVIRONMENT="CPU_COUNT=6
-                        OMP_NUM_THREADS=2
-                        OPENBLAS_NUM_THREADS=2
+    - CIBW_ENVIRONMENT="CPU_COUNT=2
+                        OMP_NUM_THREADS=1
+                        OPENBLAS_NUM_THREADS=1
```

Comment:
This seemed to be the issue. From https://travis-ci.community/t/nproc-reports-32-cores-on-arm64/5851 I think we only have access to two cores.

Reply:
I am pretty sure that there are more because I tuned those values to get significant speed-ups

Furthermore this should not trigger a crash, unless the memory is really too limited. And this configuration is already present on main which does not crash...